### PR TITLE
meeting creation form fully functional

### PR DESCRIPTION
**Pull request by JakubC-projects**

### CODE REVIEW

File: client/src/util/churches.js [hunk 1]
```diff
@@ -0,0 +1,226 @@
+export default [
```
Comment (philipdalen) [inline client/src/util/churches.js:1]: I would say this is fine for now, but i suggest that in the future this gets refactored to get the list of churches directly from members.bcc.no ideally through a web components which could live in members. That way we could start a service oriented architecture, or what do you think @StevenMalaihollo

File: client/src/views/Create.vue [hunk 2]
```diff
@@ -30,39 +30,55 @@
...
       </date-picker>
     </div>
-    <div class="horizontal padding-vertical-20">
-      <input type="checkbox" class="margin-horizontal-10" v-model="publicVoting"/>
-      <label class="medium-text">Public Results</label>
-    </div>
     <div class="horizontal width-100 space-out">
     <h2>Invited Voters</h2>
-    <h2 v-if="numOfVoters">Found {{numOfVoters}} Voters</h2>
+    <h2>Found {{numOfVoters}} People</h2>
```
Comment (philipdalen) [inline client/src/views/Create.vue:35]: Perhaps "Members" instead of "People" ?

File: client/src/util/churches.js (hunk 1 above)
Comment (StevenMalaihollo) [reply client/src/util/churches.js:1]: I will have to look into serving components from the server and if it is valuable. Styling and layout is often page specific.

File: server/src/services/members/members.class.ts [hunk 3]
```diff
@@ -38,58 +38,39 @@ export class Members implements ServiceMethods<Data> {
...
+
+    url += "&churchID="+churchID;
+
+    // if(minAge)
+    //   url += "&age[$gt]="+minAge;
+
+    // if(maxAge)
+    //   url += "&age[$lt]="+maxAge;
+
+    console.log(url);
```
Comment (StevenMalaihollo) [inline server/src/services/members/members.class.ts:67]: Remove all console logs and unneeded variabeles that we won't use in the future.

**Review by StevenMalaihollo — APPROVED** [review]

**Pull request merged**